src/main/kotlin/com/github/minecraftschurlimods/helperplugin/Util.kt

```diff
@@ -24,18 +20,17 @@ import org.gradle.kotlin.dsl.the
 import org.gradle.language.jvm.tasks.ProcessResources
 
 // TODO: remove this once https://github.com/gradle/gradle/issues/23572 is fixed
-fun Project.localGradleProperty(name: Provider<String>): Provider<String> = name.map(TransformerUtils.guard {
-    return@guard if (hasProperty(it)) property(it)?.toString() else null
-})
+fun Project.localGradleProperty(name: Provider<String>): Provider<String> = name
+    .filter { hasProperty(it) && property(it) != null }
```

Comment:
Note: Incubating API, but MDG uses it internally anyway and I don't see a good alternative that preserves nullability safety. (Otherwise we'd need a `Provider<String?>`).